go.sh

```diff
@@ -20,5 +20,36 @@
 #    files (e.g. signed public releases should match signed intel releases).
 #    "publish" rule. 
 
-echo "Nothing implemented"
+ROOT=/
+INTEL_PATH=lib/firmware/intel
+VERSION=v1.5
```

Comment:
I had a quick look at merging this into master (which would store all global and release agnostic files) and the problem is having the hard coded version here.

Could you update this to take VERSION from the git branch name i.e. search for vX.Y(.Z) in the branch name and use this for VERSION. This would then allow us to use a single script for all releases.

Reply:
In that case would it make more sense to submit the more flexible script to the master branch first and then cherry pick it to v1.5? From the README.md:

> The master branch will not contain any binaries, but will contain all the tools necessary for release management.

Reply:
@lgirdwood what if someone uses githubs download zip feature? That would also make it a lot more difficult to package this as someone would have to manually patch this (as it won't be running in git)?

Reply:
Items 3. and 4. already require git anyway. Not sure about 1., 2. and 5. but starting from a git clone doesn't seem like a very demanding requirement for a release script. If it is there's always a way, for instance something like:
```shell
# Give precedence to the environment.
test -n "${SOF_VERSION}" || SOF_VERSION=$(git describe --match ...)
```